recipes/pychemia/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pychemia" %}
+{% set version = "0.20.6.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e86e7985b4545880db6bbb7fbb16da652dfa0763e04a591797e7e8a3ebf12690
+
+build:
+  number: 0
+  skip: true   # [py>=40]
```

Comment:
What's the reason for this skip?

Reply:
I used https://github.com/marcelotrevisani/grayskull to generate the recipe, but I can remove it.